Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt

```diff
@@ -70,13 +68,16 @@ typealias CalendarWeek = List<CalendarDay>
 fun Calendar(
     calendarYear: CalendarYear,
     onDayClicked: (CalendarDay, CalendarMonth) -> Unit,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues()
 ) {
-    LazyColumn(modifier) {
+    LazyColumn(
+        modifier = modifier,
```

Comment:
I'm not sure I understand the removal of the insets spacer here.

The `contentPadding` from the `Scaffold` could be from a top or bottom bar, and can exist independently from the insets.

However, it is important that insets aren't double applied. To that end, I think we should mark the contentPadding as a consumption of the insets, and then keep the `Spacer` below: 
```suggestion
        modifier = modifier.consumedWindowInsets(contentPadding),
```

Reply:
I did test it out and it looked like it was working. I will try out this solution as well.

Reply:
Fixed, could you review again please? I wasn't sure if I still needed the contentPadding below or not